### PR TITLE
feat: e11y service account integration api

### PR DESCRIPTION
**Pull request by fredmaggiowski**

### CODE REVIEW

File: packages/console-client/src/kiota-client/api/marketplace/index.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export interface MarketplaceRequestBuilderGetQueryParameters {
 /**
  * Uri template for the request builder.
  */
-export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace{?includeTenantId*,itemUrns*,perPage*,publishOnMiaDocumentation*,resolveResourcesData*,tenantId*,types*}";
+export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace";
```
Comment (danielemarostica) [inline packages/console-client/src/kiota-client/api/marketplace/index.ts:66]: i'm not sure what is causing this, but i would leave it as was before

```suggestion
export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace{?includeTenantId*,itemUrns*,perPage*,publishOnMiaDocumentation*,resolveResourcesData*,tenantId*,types*}";
```

Comment (fredmaggiowski) [reply packages/console-client/src/kiota-client/api/marketplace/index.ts]: agree

File: packages/console-client/oas-schema/console-apis-schema.json [hunk 2]
```diff
@@ -427,6 +427,9 @@
                                     "items": {
                                         "additionalProperties": false,
                                         "properties": {
+                                            "_id": {
```
Comment (guidozoli) [inline packages/console-client/oas-schema/console-apis-schema.json:430]: it is correct to add `_id` field?

File: packages/console-client/oas-schema/console-apis-schema.json [hunk 3]
```diff
@@ -1087,6 +1200,7 @@
                                         }
                                     },
                                     "required": [
+                                        "_id",
```
Comment (guidozoli) [inline packages/console-client/oas-schema/console-apis-schema.json:1203]: it is correct to add _id field?

File: packages/console-client/oas-schema/console-apis-schema.json [hunk 4]
```diff
@@ -928,6 +989,9 @@
                                 "schema": {
                                     "additionalProperties": false,
                                     "properties": {
+                                        "_id": {
```
Comment (guidozoli) [inline packages/console-client/oas-schema/console-apis-schema.json:992]: it is correct to add _id field?

File: packages/console-client/oas-schema/console-apis-schema.json [hunk 5]
```diff
@@ -586,6 +638,7 @@
                                             }
                                         },
                                         "required": [
+                                            "_id",
```
Comment (guidozoli) [inline packages/console-client/oas-schema/console-apis-schema.json:641]: it is correct to add _id field?

**Review by danielemarostica — APPROVED** [review]

File: packages/console-client/src/kiota-client/api/extensibility/tenants/item/extensions/index.ts [hunk 6]
```diff
@@ -174,11 +192,13 @@ export function deserializeIntoExtensions(extensions: Partial<Extensions> | unde
         "entry": n => { extensions.entry = n.getStringValue(); },
         "extensionId": n => { extensions.extensionId = n.getStringValue(); },
         "iconName": n => { extensions.iconName = n.getStringValue(); },
+        "_id": n => { extensions.id = n.getStringValue(); },
```
Comment (guidozoli) [inline packages/console-client/src/kiota-client/api/extensibility/tenants/item/extensions/index.ts:195]: it is correct to add _id field?

File: packages/console-client/src/kiota-client/api/marketplace/index.ts (hunk 1 above)
Comment (guidozoli) [inline packages/console-client/src/kiota-client/api/marketplace/index.ts]: why this is changed?

**Pull request merged**